### PR TITLE
improve API docs

### LINKED ISSUE #216: Python API docs are broken
At the time as I create this issue, our API [docs](https://deltares.github.io/Ribasim/python/reference/Basin.html#ribasim.Basin) look like this:

image

I haven't checked yet, but I think that the latest quartodoc [release broke](https://github.com/machow/quartodoc/releases/tag/v0.3.0) that.

Tasks:

- [ ] Investigate whether moving back to 0.2.2 fixes things
- [ ] Pin the dependency
- [ ] Revert https://github.com/Deltares/Ribasim/pull/201

### PR DESCRIPTION
**Pull request by visr**
Fix #216.

The rest of the docstrings still need some more treatment to follow numpydoc better.

https://numpydoc.readthedocs.io/en/latest/example.html
https://machow.github.io/quartodoc/get-started/docstring-examples.html

```
python -m numpydoc --validate ribasim.model.Model
```

### CODE REVIEW

**Review by Hofer-Julian — CHANGES_REQUESTED** [review]
Great!

Can you add a linting job to ensure conformity in the CI?

**Comment by visr** [comment]
Looks like this could be used as an example for docstring validation: https://github.com/mkdocstrings/griffe/issues/104

**Comment by Hofer-Julian** [comment]
> Looks like this could be used as an example for docstring validation: [mkdocstrings/griffe#104](https://github.com/mkdocstrings/griffe/issues/104)

Looks good! I missed that you have to run `numpydoc --validate` for each symbol individually.
So let's handle that in a follow-up PR

**Comment by visr** [comment]
Yeah let's do that in a follow up. Here I can make sure things look generally ok. I'll remove the column listings in the parameter description as mentioned in https://github.com/Deltares/Ribasim/issues/216#issuecomment-1548568137.

**Review by Hofer-Julian — APPROVED** [review]
Great to have that fixed!

File: docs/_quarto.yml [hunk 1]
```diff
@@ -65,10 +65,6 @@ quartodoc:
       desc: The Model class represents an entire Ribasim model.
       contents:
         - Model
-        - Model.write
```
[the remark this answers is not in the document]
Comment (visr) [reply docs/_quarto.yml:68]: Yeah the methods are now automatically included in the class page. I think that is new in quartodoc 0.3, which is a big improvement.

**Pull request merged**